lesson08/assignment/vue-cli/src/App.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+<div id="app">
+ <div id="nav">
+    <b-navbar toggleable="lg" type="light" variant="light">
+       <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>
+       <b-collapse id="nav-collapse" is-nav>
+          <b-navbar-nav>
+             <b-nav-item  to="/" >Home</b-nav-item>
+             <b-nav-item  to="/about">About Us</b-nav-item>
+             <b-nav-item  to="/trips">Trips</b-nav-item>
+             <!--<b-nav-item  to="/test-map">Test</b-nav-item>-->
```

Comment:
Best not to leave commented out code in master. If it's something you really need later, you can retrieve it from your commit history, from the commit where it was removed. But chances are, you won't need exactly the same code later, and it'll just be clutter until deleted.